Makefile

```diff
@@ -8,7 +8,7 @@ CGO_ENABLED ?= 1
 WASM_ENABLED ?= 1
 
 # Force modules on and to use the vendor directory.
-GO := CGO_ENABLED=$(CGO_ENABLED) GO111MODULE=on GOFLAGS=-mod=vendor go
+GO := CGO_ENABLED=$(CGO_ENABLED) GO111MODULE=on GOFLAGS="-mod=vendor -buildmode=exe" go
```

Comment:
it's mentioned why we're doing this in the commit message but perhaps we should include a comment (if we squash we'll lose the context)